test/e2e/kustomize_test.go

```diff
@@ -194,3 +214,22 @@ func TestKustomizeNameSuffix(t *testing.T) {
 			assert.Contains(t, app.Spec.Source.Kustomize.NameSuffix, "-suf")
 		})
 }
+
+// make sure we we can invoke the CLI to set and unset namesuffix
+func TestKustomizeNameSuffixSetUnset(t *testing.T) {
```

Comment:
Can you merge TestKustomizeImagesSetUnset and TestKustomizeNameSuffixSetUnset into one test. The e2e tests are quite expensive. I think it this case it is better to have one test that verifies both parameters

Reply:
Two tests are merged into one. Thank you Alex for the detailed and thorough review!